fsh-generated/resources/StructureDefinition-CcaPlanDefinition.json

```diff
@@ -325,11 +325,6 @@
           }
         ]
       },
-      {
```

Comment:
wasn't sure if I just remove it and it goes back to the default or I set max to 1... anyway - went with remove